src/atoms/Trackable/utils.js

```diff
@@ -1,7 +1,7 @@
 import kebabCase from 'lodash/kebabCase';
 import assign from 'lodash/assign';
 
-export function dataAttrName(key) {
+export function dataAttrName(key = 'undefined') {
```

Comment:
What's this all about?

Reply:
@PepperTeasdale if key is `undefined` then `toLowerCase()` errors out. This will just make a `data-undefined` and move on. Not ideal.

Reply:
yeah, could we make it just not add anything instead? we were looking at some elements in gatsby earlier and there was a lot of "undefined" in class names/ data properties in the dom.